### ISSUE TITLE
Accessibility - input labels/button names/color contrast/keyboard nav

### ISSUE THREAD

**Issue by sayers24** [InstituteforDiseaseModeling/covasim#39]
- [ ] WCAG 4.1.2 - buttons must have discernible text
- [ ] WCAG 1.3.1, WCAG 3.3.2 - input controls must be associated with their labels
- [ ] WCAG 1.4.3 - contrast ratio between text and background must meet AA standards
- [ ] Keyboard navigation - tooltip information should be accessible for non-mouse users

**Comment by sayers24** [comment]
PR: Clean up styles/address accessibility issues #40

**Comment by ckerr-IDM** [comment]
Closed by #40
